tests/testthat/test-model-compile.R

```diff
@@ -275,7 +275,7 @@ test_that("compiling stops on hyphens in stanc_options", {
 test_that("compiling works with only names in list", {
   skip_on_cran()
   stan_file <- testing_stan_file("bernoulli")
-  mod <- cmdstan_model(stan_file, stanc_options = list("allow-undefined", "warn-pedantic"), force_recompile = TRUE)
+  mod <- cmdstan_model(stan_file, stanc_options = list("warn-pedantic"), force_recompile = TRUE, quiet = FALSE)
```

Comment:
allow-undefined now expects an external file (it didnt in 2.25 which was a bug) so we cant use it in a test.